sys/net/gnrc/network_layer/sixlowpan/iphc/gnrc_sixlowpan_iphc.c

```diff
@@ -92,6 +96,12 @@
 #define NHC_UDP_8BIT_PORT           (0xF000)
 #define NHC_UDP_8BIT_MASK           (0xFF00)
 
+/* currently only used with forwarding output, remove guard if more debug info
+ * is added */
+#ifdef MODULE_GNRC_SIXLOWPAN_FRAG_VRB
```

Comment:
```suggestion
#if defined(MODULE_GNRC_SIXLOWPAN_FRAG_VRB) && ENABLE_DEBUG
```

Reply:
Nope, that will lead to compilation errors, as `addr_str` is also "used" with `ENABLE_DEBUG=0` (`DEBUG` is always compiled in, but thrown out by the optimizer, as is this string array, as it is not used in this case). This is the case for quite a while now. See https://github.com/RIOT-OS/RIOT/pull/5166.